client/src/components/Search.js

```diff
@@ -26,15 +26,15 @@ const Search = ({ handleSearch }) => {
 						value={searchQuery}
 						onChange={handleChange}
 					/>
-					<img
+					{/* <img
```

Comment:
Don't comment out, delete! 😄 

Why?
To be tidy 🧹 

But also because you should always be moving forward. If you've decided you don't need this code anymore it is fine to remove it. You have the git record so you can always go back to it if you need to.